contents/swift/welcome-to-swift.md

```diff
@@ -62,14 +65,14 @@ func yearAlbumReleased(name: String) -> Int {
     case "Red": return 2012
     case "1989": return 2014
     default:
-        return -1
+        return -1 // Special value, but is not meaningful to other developers
     }
 }
 ```
 Without Optionals, you might consider using -1 to indicate that there was no such album. But if someone else uses this function, how does he know that -1 means "no such album"? Wouldn't it be better if we could return nil?
```

Comment:
Try not to start a sentence with 'But'. Shall we use 'However' here instead?

Rephrase to:
'However, if someone else uses this function, how would he know that -1 means "no such album"?`